asset-transfer-ledger-queries/chaincode-go/asset_transfer_ledger_chaincode.go

```diff
@@ -94,43 +98,42 @@ type asset struct {
 // QueryResult structure used for handling result of query
 type QueryResult struct {
 	Record              *asset
-	TxId                string    `json:"txId"`
+	TxId                string    `json:"txID"`
 	Timestamp           time.Time `json:"timestamp"`
 	FetchedRecordsCount int       `json:"fetchedRecordsCount"`
 	Bookmark            string    `json:"bookmark"`
 }
 
-// CreateAsset - create a new asset, store into chaincode state
+// CreateAsset initializes a new asset in the ledger
 func (t *SimpleChaincode) CreateAsset(ctx contractapi.TransactionContextInterface, ID, color, owner string, size, appraisedValue int) error {
```

Comment:
we're going to standardize on this order of arguments for every function:
assetID string, color string, size int, owner string, appraisedValue int

(we're choosing logical order and consistency over grouping by data type).

Go ahead and update here and in the sample invokes above.

Reply:
Done